src/universal-app/kitchen-sink-mdc/kitchen-sink-mdc.html

```diff
@@ -104,10 +104,10 @@ <h2>MDC slide-toggle</h2>
 <mat-slide-toggle>with a label</mat-slide-toggle>
 
 <h2>MDC Slider</h2>
-<mat-slider></mat-slider>
+<!-- <mat-slider></mat-slider>
 <mat-slider value="50"></mat-slider>
 <mat-slider tickInterval="1" min="1" max="10" value="5" thumbLabel></mat-slider>
-<mat-slider disabled></mat-slider>
+<mat-slider disabled></mat-slider> -->
```

Comment:
In general you should omit any commented code until it's ready to go in for real